cobc/cobc.h

```diff
@@ -146,6 +146,7 @@ enum cb_current_date {
 #define	CB_CS_DEFAULT			CB_CS_DAY
 #define	CB_CS_VALIDATE_STATUS		CB_CS_DAY
 #define	CB_CS_SPECIAL_NAMES		CB_CS_DAY
+#define	CB_CS_USAGE				CB_CS_DAY
```

Comment:
this reminds me of a 4.x TODO: please add "rework the context-sensitive reserved words handling" (or use a bigger type) to that list

Reply:
Added to the TODO list.